cmd/podmanV2/containers/unpause.go

```diff
@@ -27,6 +27,10 @@ var (
 )
 
 func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
+		Command: unpauseCommand,
+	})
 	registry.Commands = append(registry.Commands, registry.CliCommand{
 		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
 		Command: unpauseCommand,
```

Comment:
nit, it would good to switch the order of lines 36 and 37 to match the other commands.